static/js/containers/VideoDetailPage.js

```diff
@@ -85,41 +125,50 @@ class VideoDetailPage extends React.Component {
-          <a className="collection-link" href={collectionUrl}>
-            {video.collection_title}
-          </a>
-        </p>
-        <h2 className="video-title mdc-typography--title">
-          {video.title}
-        </h2>
-        { video.description
-          ?  <p className="video-description mdc-typography--body1">
-            {video.description}
-          </p>
-          : null }
-        <span className="upload-date mdc-typography--subheading1 fontgray">
-          Uploaded {formattedCreation}
-        </span>
-        <span className="actions">
-          {
-            editable &&
-            <Button
-              className="edit mdc-button--raised"
-              onClick={showDialog.bind(this, DIALOGS.EDIT_VIDEO)}
-              disabled={!editable}
-            >
-              <span className="material-icons">edit</span> Edit
-            </Button>
-          }
-          <Button
-            className="share mdc-button--raised"
-            onClick={showDialog.bind(this, DIALOGS.SHARE_VIDEO)}
-          >
-            <span className="material-icons">share</span> Share
-          </Button>
-        </span>
+      <div className="mdc-layout-grid">
+        <div className="mdc-layout-grid__inner">
+          <div className="summary mdc-layout-grid__cell--span-8">
+            <p className="channelLink mdc-typography--subheading1">
+              <a className="collection-link" href={collectionUrl}>
+                {video.collection_title}
+              </a>
+            </p>
+            <h2 className="video-title mdc-typography--title">
+              {video.title}
+            </h2>
+            { video.description
+              ?  <p className="video-description mdc-typography--body1">
+                {video.description}
+              </p>
+              : null }
+            <span className="upload-date mdc-typography--subheading1 fontgray">
+              Uploaded {formattedCreation}
+            </span>
+            <span className="actions">
+              {
+                editable &&
+                <Button
+                  className="edit mdc-button--raised"
+                  onClick={showDialog.bind(this, DIALOGS.EDIT_VIDEO)}
+                  disabled={!editable}
+                >
+                  <span className="material-icons">edit</span> Edit
+                </Button>
+              }
+              <Button
+                className="share mdc-button--raised"
+                onClick={showDialog.bind(this, DIALOGS.SHARE_VIDEO)}
+              >
+                <span className="material-icons ">share</span> Share
+              </Button>
+            </span>
+          </div>
+          <div className="videoSubtitles mdc-layout-grid__cell--span-4">
+            <VideoSubtitleCard id="subtitleCard" video={video} isAdmin={editable}
+              uploadVideoSubtitle={this.setUploadSubtitle}
+              deleteVideoSubtitle={this.deleteSubtitle}/>
```

Comment:
As mentioned above, I think we generally prefer new lines for components w/ lots of attributes, but we can discuss if you have a strong preference the other way

Reply:
OK